go.mod

```diff
@@ -134,6 +134,4 @@ replace (
 	go.opentelemetry.io/otel/trace => go.opentelemetry.io/otel/trace v0.20.0
 	go.opentelemetry.io/proto/otlp => go.opentelemetry.io/proto/otlp v0.7.0
 
-	// Required because of cert-manager's outdated gateway API import.
-	sigs.k8s.io/gateway-api => sigs.k8s.io/gateway-api v0.4.3
```

Comment:
Gateway API was updated in cert-manager so this is no longer needed: 
 * https://github.com/cert-manager/cert-manager/pull/5376